modules/tinymce/package.json

```diff
@@ -37,6 +37,8 @@
   },
   "devDependencies": {
     "@rollup/plugin-alias": "^5.1.1",
-    "@rollup/plugin-node-resolve": "^16.0.1"
+    "@rollup/plugin-node-resolve": "^16.0.1",
+    "eslint-plugin-oxlint": "^0.16.3",
+    "oxlint": "^0.16.3"
```

Comment:
So we are only using oxlint for the tinymce project not the other projects? I would prefer that we run the same lint rules on all code in the project and have some config to disable annoying rules per project if needed. So could this be moved up.

Reply:
See my other long comment [here](https://app.graphite.dev/github/pr/tinymce/tinymce/10292/TINY-11629-Upgrade-the-tinymce-repo-to-eslint-v9#comment-PRRC_kwDOAAciD856hu3S)